.vscode/launch.json

```diff
@@ -5,19 +5,15 @@
   "version": "0.2.0",
   "configurations": [
     {
-      // Has convenient settings for attaching to a NodeJS process for debugging purposes
-      // that are NOT the default and otherwise every developers has to configure for
-      // themselves again and again.
       "type": "node",
-      "request": "attach",
-      "name": "Attach to NodeJS",
-      // If we don't do this, every step-into into an async function call will go into
-      // NodeJS internals which are hard to step out of.
-      "skipFiles": [
-        "<node_internals>/**"
-      ],
-      // Saves some button-pressing latency on attaching
-      "stopOnEntry": false
+      "request": "launch",
+      "args": ["jest", "--runTestsByPath", "/Users/bergjak/workplace/CDK/aws-cdk/packages/aws-cdk-lib/aws-lambda-nodejs/test/function.test.ts"],
+      "name": "debug unit test",
+      "program": "/opt/homebrew/bin/yarn",
+      "cwd": "/Users/bergjak/workplace/CDK/aws-cdk/packages/aws-cdk-lib",
+      "console": "integratedTerminal",
+      "sourceMaps": true,
+      "skipFiles": [ "<node_internals>/**/*" ],
```

Comment:
leftover?

Reply:
yeah, I'll remove this in my final revision